nixos/tests/all-tests.nix

```
@@ -198,6 +198,9 @@ in
   haka = handleTest ./haka.nix {};
   haproxy = handleTest ./haproxy.nix {};
   hardened = handleTest ./hardened.nix {};
+  hbase1 = handleTest ./hbase.nix { package=pkgs.hbase1; };
+  hbase2 = handleTest ./hbase.nix { package=pkgs.hbase2; };
+  hbase3 = handleTest ./hbase.nix { package=pkgs.hbase3; };
```

```suggestion
  hbase1 = handleTest ./hbase.nix { package = pkgs.hbase1; };
  hbase2 = handleTest ./hbase.nix { package = pkgs.hbase2; };
  hbase3 = handleTest ./hbase.nix { package = pkgs.hbase3; };
```
and they should be in passthru.